response_types.go

```diff
@@ -36,7 +36,7 @@ type FeaturesResponse struct {
 	Langauges []FeatureLanguage `json:"languages"`
 }
 
-// TestPayload is a single test for a Module
+// TestPayload is a single test for a Plugin
```

Comment:
```suggestion
// TestPayload is a single test for a plugin
```
Nit based on feedback in other repos.